engine/src/mac-qt-player.mm

```diff
@@ -167,6 +227,8 @@ inline NSComparisonResult do_QTTimeCompare (QTTime time, QTTime otherTime)
 {
 	m_movie = [[NSClassFromString(@"QTMovie") movie] retain];
 	m_view = [[NSClassFromString(@"QTMovieView") alloc] initWithFrame: NSZeroRect];
+    m_helper = [[com_runrev_livecode_MCQTKitHelper alloc] init];
+    [m_helper dynamicallySubclassQTMovieView:m_view];
```

Comment:
Because dynamicallySubclassQTMovieView is now a class method you can do:
```
[com_runrev_livecode_MCQTKitHelper dynamicallySubclassQTMovieView: m_view];
```